HW16/Team.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <vector>
+#include <string>
+#include "Player.h"
```

Comment:
Team includes Player, and Player includes Team. I wonder why it didn't break up)